### ISSUE TITLE
UI improvement: "Show Statistics" button should be more visible 

### ISSUE THREAD

**Issue by glynhudson** [emoncms/graph#36]
The "Show Statistics" tag needs to be more prominent.

- Make text darker (black?)
- Make tab action more prominent

![](https://user-images.githubusercontent.com/758844/48382171-b5f6af00-e6d7-11e8-8203-a98db53ae676.png)

Extension of https://github.com/emoncms/graph/issues/35

**Comment by emrysr** [comment]
Might be better to add a tab navigation here to jump between the two views?
[mock up of 2 tab layout]
![graph-issue-36-mockup](https://user-images.githubusercontent.com/1466013/48428357-c011bf80-e762-11e8-84e6-016829cea085.png)

**Comment by TrystanLea** [comment]
Would using black font colour be sufficient?

**Comment by glynhudson** [comment]
We did look test with a black font, however it just looked like a table header rather than a clickable tab hyperlink. The tab suggestion by @emrysr would make this very clear.

**Comment by TrystanLea** [comment]
Keen for the flat styling to be in keeping with rest of the site, any other bootstrap v2 options that might work @emrysr ?

**Comment by emrysr** [comment]
discussed with @TrystanLea and @glynhudson.. simplest fix is to use the built in bootstrap 2.3 class names for buttons:
![image](https://user-images.githubusercontent.com/1466013/51253505-62823500-1996-11e9-99b0-e9ab67d84dd2.png)

the classes `btn` and `btn-default` (and some margining) will make the button look like the above screenshot

pasted edits from inspector
```
<div class="feed-options-show-options hide btn btn-default" style="display: none;">Show options</div>
<div class="feed-options-show-stats btn btn-default" style="display: block;">Show statistics</div>
```

css changes from: 
```
.feed-options-show-stats {
    float: right;
    padding: 10px;
    width: 150px;
    text-align: center;
    border-left: 1px solid #eee;
}
```
to: 
```
.feed-options-show-stats {
    float: right;
    margin: 5px;
}
```
